### ISSUE TITLE
Help on interpreting results - strenght of temporal signal

### ISSUE THREAD

**Issue by andreaniml** [xavierdidelot/BactDating#50]
Hello! I'm new to BactDating and I'm trying to run it on a phylogeny of 435 samples of _Salmonella_. The phylogeny was generated by creating a snp alignment using Snippy and passing this to Gubbins.

I got the dates from NCBI and wanted to compare BactDating with TempEst regarding temporal signal. Currently I ran into some issues (I think).
My workflow:
1)As NCBI sometimes has only the year specified and sometimes  year-month-day, for creating the decimal dates format I copied how TempEst imported the dates (example: a year turned from 2015 to 2015.0)
2) I generated the dates vector by importing the table with "tip \t date" information and used d = c(Table$dates) and names(d) = c(Table$tips)
3)  Then I generated the analysis and a set of root to tip graphics:
t = LoadGubbins(prefix ='my_prefix')
rooted=initRoot(t,d)
r2=roottotip(t,d)
r2=roottotip(rooted,d)
res=bactdate(t,d)
plot(res,'treeCI')
plot(res,'trace')

Unrooted tree:
![image](https://user-images.githubusercontent.com/58482628/137990675-4622175a-02aa-4afe-a5d0-59e0915e5e62.png)
Rooted Tree:
![image](https://user-images.githubusercontent.com/58482628/137990876-039e5076-3085-4b27-bbb9-45d7191e2c66.png)
trace:
![image](https://user-images.githubusercontent.com/58482628/137991191-bdc53996-a439-4455-988b-fbdb4a036011.png)
treeCI:
![image](https://user-images.githubusercontent.com/58482628/137991364-bb8658e6-c37a-4ec7-a260-7c1060d12970.png)

So, here are my concerns:

1)Apparently I do have significant signal (the p value is low) but the line is too flat, my R is below 0.00. Both in rooted and unrooted trees. I do have two outliers (one is obvious from the BactDating plot, the other is only visible on TempEst)
2) My trace seems a little thin, but I think that given the root to tip graphics results I would rather change my dataset a little than run it longer
3)My TreeCI output does not show the years correctly on the bar, I'm wondering If I actually messed up on my input (why is it backwards? Do I have to insert a backwards = False flag somewhere and forgot?)

Thanks!

**Comment by andreaniml** [comment]
Thanks! I've removed the genome and the analysis is running!
Just an aditional quick question: I've seen on the vignetes that you should rescale the tree if  the branch lenghts are measured per site. Whenever I run an analysis on gubbins, I can generate a tree with support or without it.  If I choose the SH support sometimes it rescales to "substitutions per site", I would have to do the t$edge.length=t$edge.length*L thing to use the sh supported tree, even if it is an input from Gubbins, right? (or simply avoid using sh supported tree altogether and do a simple 100 bootstrap tree instead?)

**Comment by xavierdidelot** [comment]
Yes that's right. If you can get Gubbins to output the tree in substitutions rather than substitutions per site then this would probably be the easiest and best.
